src/Microsoft.Crank.AzureDevOpsWorker/Program.cs

```diff
@@ -48,15 +57,64 @@ public static int Main(string[] args)
                     queue = Environment.GetEnvironmentVariable(queue);
                 }
 
-                await ProcessAzureQueue(connectionString, queue);
+                if (certBasedAuth.HasValue() && !(certClientId.HasValue() && certTenantId.HasValue() && certThumbprint.HasValue()))
```

Comment:
This doesn't handle the case where someone passes a cert path instead of a thumbprint (not sure if there is a need for both to be configured at once). Is the goal of having certPath as an option to make it so that we can rotate certificates without having to modify the args passed to crank?

Reply:
I added it because I was wanting to make sure that we were giving maximum flexibility in how people could use the tool. Good catch on missing that case. I have updated the expression so that it should do what we want.